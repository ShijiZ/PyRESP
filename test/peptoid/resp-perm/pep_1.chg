All values are reported in atomic units
%FLAG TITLE
 molecule 1:  peptoid

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
   2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
   3   7.1693530E+00   3.8731120E-01   1.6170360E-01
   4   5.4836410E+00   4.7919210E-01   3.0292610E+00
   5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
   6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
   7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
   8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
   9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
  10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
  11   1.5824870E+00   3.9493260E+00   1.3602580E+00
  12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
  13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
  14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
  15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
  16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
  17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
  18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
  19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
  20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
  21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
  22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          1          0     6.7506048E-02
   2          6          0     2.7540593E-02
   3          1          0     6.8713458E-02
   4          1          0     5.8767596E-02
   5          6          0     3.5764807E-01
   6          8          0    -5.8017576E-01
   7          7          0    -3.0506866E-01
   8          6          0    -2.1829418E-02
   9          1          0     6.2146655E-02
  10          1          0     3.4603157E-02
  11          1          0     7.3451441E-02
  12          6          0    -1.0194315E-02
  13          1          0     1.7544590E-01
  14          1          0     8.3884235E-02
  15          6          0     4.3182782E-01
  16          8          0    -5.2426683E-01
  17          7          0    -3.7569131E-01
  18          1          0     1.3589016E-01
  19          6          0    -2.1112955E-02
  20          1          0     1.8659308E-01
  21          1          0     6.1734172E-02
  22          1          0     1.2586847E-02

%FLAG PERM DIP LOCAL: 3(I4,X5),I4,X2,E16.7
 dip.no   atm.no   ref.no   ivary      p(opt)
   1        1        2        0    -2.8273177E-02
   2        2        1        0    -5.6571255E-02
   3        2        3        0    -1.8315298E-03
   4        2        4        0     1.8273583E-02
   5        2        5        0     3.2828629E-02
   6        3        2        0    -2.3354166E-02
   7        4        2        0    -3.2968677E-02
   8        5        2        0    -4.4043041E-04
   9        5        6        0    -1.1612538E-01
  10        5        7        0     1.2800897E-01
  11        6        5        0     3.2605223E-01
  12        7        5        0    -1.7361300E-02
  13        7        8        0     3.4226996E-02
  14        7       12        0    -3.2653824E-03
  15        8        7        0    -1.2994470E-01
  16        8        9        0     5.8176585E-03
  17        8       10        0     4.6523785E-02
  18        8       11        0     3.6099649E-02
  19        9        8        0    -2.9480694E-02
  20       10        8        0    -6.8303959E-02
  21       11        8        0    -1.3602796E-02
  22       12        7        0    -5.8024616E-02
  23       12       13        0     6.7415144E-03
  24       12       14        0    -3.5379571E-02
  25       12       15        0     1.1879510E-01
  26       13       12        0     3.4723989E-02
  27       14       12        0    -6.2485987E-02
  28       15       12        0     2.1323313E-02
  29       15       16        0    -1.2729841E-01
  30       15       17        0     8.4827352E-02
  31       16       15        0     3.4103186E-01
  32       17       15        0    -9.0013792E-02
  33       17       18        0     8.0054872E-02
  34       17       19        0     1.4907537E-02
  35       18       17        0    -3.3180727E-01
  36       19       17        0    -6.3038390E-02
  37       19       20        0    -8.0434043E-04
  38       19       21        0     1.3623947E-02
  39       19       22        0     4.4312562E-02
  40       20       19        0     5.8309024E-02
  41       21       19        0    -3.5964198E-03
  42       22       19        0    -4.7501826E-02

%FLAG PERM DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   1.1538575E-02  -2.4671215E-02   7.5871598E-03
   2  -5.5459244E-02   5.1682492E-02  -1.5200540E-02
   3   1.4946635E-02   1.1688650E-02  -1.3615823E-02
   4  -6.0516980E-03   1.7978629E-02   2.6964412E-02
   5  -2.9953091E-02   2.0831308E-01  -3.1182134E-02
   6   1.0226533E-01   3.0644699E-01   4.4069302E-02
   7   8.6746721E-05   4.6840159E-02   2.9545389E-03
   8  -3.3494319E-03   1.6121415E-01   2.5981597E-02
   9   2.8872827E-02   4.6982395E-03   3.6602866E-03
  10  -2.1046704E-02   3.9422087E-02  -5.1656230E-02
  11  -6.7333067E-03   3.4875142E-03   1.1293178E-02
  12  -1.3875022E-01   2.7350559E-02   7.1287911E-02
  13   7.9116821E-03  -2.3390489E-02   2.4414048E-02
  14   1.6979889E-03  -5.7032960E-02  -2.5472669E-02
  15  -8.4137371E-02  -6.4590320E-02  -1.6128708E-01
  16  -6.1316157E-02  -1.0491540E-01  -3.1864685E-01
  17  -1.0123246E-01  -5.4327826E-02  -9.4643956E-02
  18  -3.4726733E-02  -1.1379839E-01  -3.0974190E-01
  19  -7.5149406E-02  -3.5503140E-02   3.7756669E-02
  20  -1.6930841E-02  -2.0942217E-02  -5.1717622E-02
  21  -2.6593786E-03   2.0400795E-03  -1.3038469E-03
  22  -1.5776685E-02  -4.4456004E-02   5.5842050E-03

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1  -9.5833630E-03  -1.6826873E-02  -3.6832748E-03
   2  -1.4294105E-01  -4.4461324E-02  -7.9049226E-02
   3  -1.1404545E-02   9.1469447E-03  -1.5774748E-02
   4  -1.9011615E-02   8.9869500E-03  -1.5145635E-03
   5  -3.1503717E-01  -4.3292348E-01  -1.5759901E-01
   6  -8.6521085E-02  -2.2843094E-01  -3.7338937E-02
   7   7.3996203E-02  -9.4485124E-02   6.2156625E-02
   8  -4.5052001E-02  -1.6350359E-01  -4.6798866E-02
   9  -1.0440139E-02  -9.6843876E-03  -6.4838274E-03
  10   1.8240235E-03  -1.3666095E-02   3.8933924E-03
  11  -9.1228527E-03  -1.5534683E-02  -5.9939732E-03
  12   9.8804073E-02  -5.5100948E-02   1.5664229E-01
  13   1.2064826E-03  -1.1574027E-03   1.9786272E-02
  14   1.1491958E-03  -5.9719836E-03   2.0645944E-02
  15   1.7549716E-02   1.0894724E-01   4.4390980E-01
  16   8.0662916E-02   7.2701957E-02   2.0689062E-01
  17   1.0785445E-01   1.1626612E-02  -3.0089567E-02
  18  -5.4739904E-03   2.4601285E-02   9.1622814E-02
  19   1.1219417E-01  -6.0090502E-02  -1.8572312E-01
  20   1.4689966E-02  -4.5913530E-03  -1.4484886E-02
  21   7.4283607E-03  -7.1477826E-03  -1.9149263E-02
  22   7.5737712E-03  -7.6284263E-03  -1.8795337E-02

