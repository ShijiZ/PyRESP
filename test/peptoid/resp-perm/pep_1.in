 first stage of RESP. Peptoid for resp charge calculations
 &cntrl 
 nmol = 1,
 iqopt = 1,
 ihfree = 1,
 qwt = 0.0005,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 1,
 pwt = 0.0005,
 virtual = 0 
 &end 
  1.0 
 peptoid
    0   22 42
    1    0  0
    6    0  0  0  0  0
    1    0  0
    1    0  0
    6    0  0  0  0
    8    0  0
    7    0  0  0  0
    6    0  0  0  0  0
    1    0  0
    1    0  0
    1    0  0
    6    0  0  0  0  0
    1    0  0
    1    0  0
    6    0  0  0  0
    8    0  0
    7    0  0  0  0
    1    0  0
    6    0  0  0  0  0
    1    0  0
    1    0  0
    1    0  0
    6   0.0
    1    1    1    2    1    3    1    4    1    5    1    6 
    6   0.0
    1   17    1   18    1   19    1   20    1   21    1   22 



