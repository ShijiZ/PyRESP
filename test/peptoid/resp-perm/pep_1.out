
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
  first stage of RESP. Peptoid for resp charge calculations
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0
 ipermdip    = 1   pwt         = 0.00050000
 virtual     = 0

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name:  peptoid
 Total charge (ich):  0
 Number of centers: 22
 Number of permanent dipoles: 42
    1    1    0    0
    2    6    0    0    0    0    0
    3    1    0    0
    4    1    0    0
    5    6    0    0    0    0
    6    8    0    0
    7    7    0    0    0    0
    8    6    0    0    0    0    0
    9    1    0    0
   10    1    0    0
   11    1    0    0
   12    6    0    0    0    0    0
   13    1    0    0
   14    1    0    0
   15    6    0    0    0    0
   16    8    0    0
   17    7    0    0    0    0
   18    1    0    0
   19    6    0    0    0    0    0
   20    1    0    0
   21    1    0    0
   22    1    0    0
 ---------------------------------------------------------
 Intra-Molecular Charge Constraint(s) Info for Molecule 1
 ---------------------------------------------------------
    1    1    1    2    1    3    1    4    1    5    1    6
    1   17    1   18    1   19    1   20    1   21    1   22

 -----------------------------
 Atom/Dipole Ivary Information
 -----------------------------
    1    0    0
    6    0    0    0    0    0
    1    0    0
    1    0    0
    6    0    0    0    0
    8    0    0
    7    0    0    0    0
    6    0    0    0    0    0
    1    0    0
    1    0    0
    1    0    0
    6    0    0    0    0    0
    1    0    0
    1    0    0
    6    0    0    0    0
    8    0    0
    7    0    0    0    0
    1    0    0
    6    0    0    0    0    0
    1    0    0
    1    0    0
    1    0    0

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   22
 Total number of permanent dipoles      =   42
 Weight factor on initial charge restraints=  0.000500
 Weight factor on initial dipole restraints=  0.000500
 There are  3 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    22
 total number of esp points =   983

 center       X               Y               Z
    1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
    2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
    3   7.1693530E+00   3.8731120E-01   1.6170360E-01
    4   5.4836410E+00   4.7919210E-01   3.0292610E+00
    5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
    6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
    7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
    8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
    9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
   10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
   11   1.5824870E+00   3.9493260E+00   1.3602580E+00
   12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
   13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
   14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
   15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
   16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
   17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
   18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
   19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
   20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
   21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
   22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 and p0 values will be set to 0

 Number of unique UNfrozen charge centers =    22
 Number of unique UNfrozen permanent dipoles =    42

 Non-linear optimization requested.
  qchnge =    5.1930544714E-02
  qchnge =    8.6693250873E-03
  qchnge =    1.6011804890E-03
  qchnge =    7.7143945563E-04
  qchnge =    4.8549526702E-04
  qchnge =    3.0930369949E-04
  qchnge =    1.9553120691E-04
  qchnge =    1.2289287917E-04
  qchnge =    7.7130703305E-05
  qchnge =    4.8514470388E-05
  qchnge =    3.0656189404E-05
  qchnge =    1.9489837345E-05
  qchnge =    1.2475386687E-05
  qchnge =    8.0412087113E-06
  qchnge =    5.2179464736E-06
  qchnge =    3.4069297772E-06
  qchnge =    2.2367748007E-06
  qchnge =    1.4755799657E-06
  qchnge =    9.7740470833E-07

 Convergence in   18 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1:  peptoid

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         1        0.000000      0.067506      0       0.000000
    2         6        0.000000      0.027541      0       0.004821
    3         1        0.000000      0.068713      0       0.000000
    4         1        0.000000      0.058768      0       0.000000
    5         6        0.000000      0.357648      0       0.001346
    6         8        0.000000     -0.580176      0       0.000849
    7         7        0.000000     -0.305069      0       0.001557
    8         6        0.000000     -0.021829      0       0.004885
    9         1        0.000000      0.062147      0       0.000000
   10         1        0.000000      0.034603      0       0.000000
   11         1        0.000000      0.073451      0       0.000000
   12         6        0.000000     -0.010194      0       0.004974
   13         1        0.000000      0.175446      0       0.000000
   14         1        0.000000      0.083884      0       0.000000
   15         6        0.000000      0.431828      0       0.001128
   16         8        0.000000     -0.524267      0       0.000937
   17         7        0.000000     -0.375691      0       0.001286
   18         1        0.000000      0.135890      0       0.000000
   19         6        0.000000     -0.021113      0       0.004892
   20         1        0.000000      0.186593      0       0.000000
   21         1        0.000000      0.061734      0       0.000000
   22         1        0.000000      0.012587      0       0.000000

 Sum over the calculated charges:     -0.000

       Permanent Dipoles Before & After Optimization
 dip.no   element.no   p(init)        p(opt)     ivary    d(rstr)/dp
    1         1        0.000000     -0.028273      0       0.000000
    2         6        0.000000     -0.056571      0       0.004352
    3         6        0.000000     -0.001832      0       0.004999
    4         6        0.000000      0.018274      0       0.004919
    5         6        0.000000      0.032829      0       0.004751
    6         1        0.000000     -0.023354      0       0.000000
    7         1        0.000000     -0.032969      0       0.000000
    8         6        0.000000     -0.000440      0       0.005000
    9         6        0.000000     -0.116125      0       0.003263
   10         6        0.000000      0.128009      0       0.003078
   11         8        0.000000      0.326052      0       0.001466
   12         7        0.000000     -0.017361      0       0.004926
   13         7        0.000000      0.034227      0       0.004731
   14         7        0.000000     -0.003265      0       0.004997
   15         6        0.000000     -0.129945      0       0.003049
   16         6        0.000000      0.005818      0       0.004992
   17         6        0.000000      0.046524      0       0.004533
   18         6        0.000000      0.036100      0       0.004703
   19         1        0.000000     -0.029481      0       0.000000
   20         1        0.000000     -0.068304      0       0.000000
   21         1        0.000000     -0.013603      0       0.000000
   22         6        0.000000     -0.058025      0       0.004325
   23         6        0.000000      0.006742      0       0.004989
   24         6        0.000000     -0.035380      0       0.004714
   25         6        0.000000      0.118795      0       0.003220
   26         1        0.000000      0.034724      0       0.000000
   27         1        0.000000     -0.062486      0       0.000000
   28         6        0.000000      0.021323      0       0.004890
   29         6        0.000000     -0.127298      0       0.003089
   30         6        0.000000      0.084827      0       0.003813
   31         8        0.000000      0.341032      0       0.001407
   32         7        0.000000     -0.090014      0       0.003716
   33         7        0.000000      0.080055      0       0.003903
   34         7        0.000000      0.014908      0       0.004945
   35         1        0.000000     -0.331807      0       0.000000
   36         6        0.000000     -0.063038      0       0.004230
   37         6        0.000000     -0.000804      0       0.005000
   38         6        0.000000      0.013624      0       0.004954
   39         6        0.000000      0.044313      0       0.004571
   40         1        0.000000      0.058309      0       0.000000
   41         1        0.000000     -0.003596      0       0.000000
   42         1        0.000000     -0.047502      0       0.000000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          4.4905295E-01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   1.2350785E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.1209094E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         5.2444326E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        3.88829   -1.18829    2.56842   -2.66645

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   10.62277
        Y    7.85056   -8.07818
        Z    6.15956   -1.54419   -2.54459

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   15.09413
        Y    0.00000   -2.71291
        Z    0.00000    0.00000  -12.38122